devel/python/CMakeLists.txt

```diff
@@ -1,28 +1,40 @@
 include(cmake/find_python_module.cmake)
-include(cmake/python_module_versions.cmake) # finds version
+include(cmake/pycmake.cmake) # finds version
 
 FIND_PACKAGE(PythonInterp 2.7)
 if (NOT DEFINED PYTHON_EXECUTABLE)
    message("Python2.7 interpreter not found - Python wrappers not enabled")
    return()
 endif()
 
+python_module( cwrap QUIET ) # we can ship our own
 
-python_module(numpy 1.7.1)
+
+
+# numpy is needed for Python
+
+set( numpy_target_version '1.7.1')
+
+python_module( numpy OPTIONAL )
 if (NOT DEFINED PY_numpy)
-   message("numpy module not found - Python wrappers not enabled")
-   return()
+  message(WARNING "numpy module not found.  Python wrappers not enabled.")
```

Comment:
I agree this *is* a warning - semantically; but the warnings are very ugly - with a cmake backtrace we are not really interested in.